demos/canvas-renderer-matplotlib/demo-1/demo.html

```diff
@@ -0,0 +1 @@
+<script src="https://cdn.jsdelivr.net/pyodide/v0.19.1/full/pyodide.js"></script>
```

Comment:
```suggestion
<script src="https://cdn.jsdelivr.net/pyodide/v0.20.0a1/full/pyodide.js"></script>
```

I can confirm that this worked.

Reply:
done, thanks